src/components/Datatable/Datatable.tsx

```diff
@@ -162,6 +162,8 @@ Datatable.propTypes = {
     }),
     defaultIsFilteringOpen: PropTypes.bool,
     defaultIsFilteringApplied: PropTypes.bool,
+    defaultIsColumnOrderingOpen: PropTypes.bool,
+    defaultIsColumnOrderingApplied: PropTypes.bool,
```

Comment:
🙏 thank you, didn't think of it but will actually need this for the persistent state

Reply:
This was already implemented, but I forgot to add it to the prop types and noticed that when I got console warnings:) 
I actually thought that we have onControlChange method but I was wrong :D we probably should add it, but again until I have some clear decision about DS future, til that I consider it in "maintenance" mode and I don't plan to do anything what's not really needed